testapp/home/migrations/0025_remove_callout_tickbox.py

```diff
@@ -1,9 +1,17 @@
 # Generated by Django 3.2 on 2022-07-11 12:26
 
 from django.db import migrations
-import wagtail.blocks
-import wagtail.blocks.field_block
-import wagtail.fields
+from wagtail import VERSION as WAGTAIL_VERSION
```

Comment:
I agreed with this change.

I looks like a new migration since the Wagtail 3 upgrade was done. Nice catch!